src/main/java/org/example/cosmocats/controller/GlobalException/GlobalExceptionHandler.java

```diff
@@ -65,4 +66,8 @@ public ResponseEntity<Object> handleGlobalException(
 
         return new ResponseEntity<>(problemDetail, HttpStatus.INTERNAL_SERVER_ERROR);
     }
+    @ExceptionHandler(FeatureToggleNotEnabledException.class)
+    public ResponseEntity<String> handleFeatureToggleNotEnabled(FeatureToggleNotEnabledException ex) {
```

Comment:
In general it considered good practice  to use Problem details https://docs.spring.io/spring-framework/reference/web/webmvc/mvc-ann-rest-exceptions.html